src/components/Footer/Footer.js

```diff
@@ -0,0 +1,58 @@
+import styles from "./Footer.module.css";
+
+function Footer() {
+  return (
+    <section className={styles.wrapper}>
+      <ul className={styles.list}>
+        <li className={styles.footerItem}>
```

Comment:
All of the elements in there are attached to a `footer.js` by `css.modules`.
You could use just listItems, but it is optional.

Reply:
ah, I understand! I will leave it as is for now.